examples/organization-level-audit-log-exclude-folders/README.md

```diff
@@ -0,0 +1,26 @@
+# Integrate GCP Organization with Lacework
+The following provides an example of integrating a Google Cloud Organization with Lacework for Cloud Audit Log analysis.
+
+```hcl
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+provider "google" {}
+
+provider "lacework" {}
+
+module "gcp_organization_level_audit_log" {
+  source               = "lacework/audit-log/gcp"
+  version              = "~> 1.0"
+  bucket_force_destroy = true
+  org_integration      = true
+  organization_id      = "my-organization-id"
+}
```

Comment:
It would be good to update this README with an example of the exclude

Reply:
Helps when you hit save before commiting.  Should be in there now, thanks for catching that!